pkg/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', 'build/x86_64_xenial' ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'run', '-v', '/workspace:/workspace', '-w', '/workspace/src/',  'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', '/bin/bash', '-c', 'make purge && make -j4 DISTRO=xenial deb' ]
+- name: 'ubuntu:xenial'
+  args: [ 'cp', '/workspace/src/stackdriver-metadata_${_METADATA_AGENT_VERSION}.xenial.deb', '/workspace/pkg/docker/stackdriver-metadata_${_METADATA_AGENT_VERSION}.xenial.deb' ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '--build-arg', 'package=stackdriver-metadata_${_METADATA_AGENT_VERSION}.xenial.deb', '-t', 'gcr.io/$PROJECT_ID/metadata-agent-xenial', '-f', 'pkg/docker/Dockerfile-xenial', 'pkg/docker' ]
+images:
+- 'gcr.io/$PROJECT_ID/metadata-agent-xenial'
```

Comment:
What worries me is that there would really be nothing in this container's tag to indicate the version of the agent, would there? How would we know what version it contains?

Reply:
Is that any better than the random hash we would get otherwise, with respect to knowing what's in the image?

Reply:
We can always retag later with a more meaningful tag. If you'd like something more specific, let's follow up in a subsequent CL to unblock the build provenance team.

Reply:
Can we please add a comment that the `$BUILD_ID` tag will allow one to access the build log, which will contain the version? That actually addresses the original concern on this thread.

Reply:
Done.